deps/openssl/openssl/fuzz/helper.py

```diff
@@ -45,7 +46,7 @@ def main():
 
     cmd = ([os.path.abspath(os.path.join(THIS_DIR, FUZZER))]  + sys.argv[2:]
            + ["-artifact_prefix=" + corpora[1] + "/"] + corpora)
-    print " ".join(cmd)
+    print(" ".join(cmd))
```

Comment:
Merged in upstream in openssl/openssl#7409 but retained here because openssl in not pip installed but is vendored in.